tests/test_parser.py

```diff
@@ -0,0 +1,41 @@
+from cwl_utils.parser import cwl_version, load_document, save
```

Comment:
This file needs a top level doc string to explain its purpose. You can check it with `make diff_pydocstyle_report`

Reply:
Done! Thanks!